src/nupic/test/abstract_temporal_memory_test.py

```diff
@@ -20,26 +20,41 @@
 # ----------------------------------------------------------------------
 
 import unittest
+from abc import ABCMeta, abstractmethod
 
 from nupic.data.generators.sequence_machine import SequenceMachine
 from nupic.research.monitor_mixin.temporal_memory_monitor_mixin import (
   TemporalMemoryMonitorMixin)
-from nupic.research.temporal_memory import TemporalMemory
 
 
 
-class AbstractTemporalMemoryTest(unittest.TestCase):
+class AbstractTemporalMemoryTest(object):
```

Comment:
@vitaly-krugl Why won't tab-completion work? `TutorialTemporalMemoryTest` inherits from `unittest.TestCase` via multiple inheritance. IDEs can handle that, right?